package.json

```diff
@@ -15,6 +15,7 @@
     "http2": "^3.3.7",
     "vue": "^2.5.2",
     "vue-image-upload-resize": "^1.1.6",
+    "vue-notification": "^1.3.16",
```

Comment:
This package allows simply add flash notifications to you page (on data saved, info, etc)

Reply:
Thank you.